tests/test_chainermn.py

```diff
@@ -0,0 +1,430 @@
+import gc
+from types import TracebackType
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Optional
+from typing import Type
+
+from optuna import create_study
+from optuna import distributions
+from optuna import integration
+from optuna import pruners
+from optuna import Study
+from optuna import TrialPruned
+from optuna.integration.chainermn import ChainerMNStudy
+from optuna.integration.chainermn import ChainerMNTrial
```

Comment:
I'm not sure, but how about testing the classes in this repository directly instead of the alias in the `optuna` package?

```suggestion
from optuna_integration.chainermn import ChainerMNStudy
from optuna_integration.chainermn import ChainerMNTrial
```